install/kubernetes/cilium/values.yaml

```diff
@@ -976,8 +976,10 @@ hubble:
       rollingUpdate:
         maxUnavailable: 1
 
+    # -- Security context to be added to Hubble UI pods
     securityContext:
-      # -- Whether to set the security context on the Hubble UI pods.
+      # -- Deprecated in favor of hubble.ui.securityContext.
+      # Whether to set the security context on the Hubble UI pods.
       enabled: true
```

Comment:
Do we want to add
```yaml
runAsUser: 1001
runAsGroup: 1001
fsGroup: 1001
```
under `securityContext` here? That way when we remove the `enabled` flag, the "default securityContext" will be the same as it is now.

Reply:
@kaworu default values updated, please have a look, thanks for the advise.